application/console/controllers/BatchController.php

```diff
@@ -2,17 +2,32 @@
 
 namespace Sil\Idp\IdSync\console\controllers;
 
+use Sentry\CheckInStatus;
 use Sil\Idp\IdSync\common\traits\SyncProvider;
+use Yii;
 use yii\console\Controller;
 
+use function Sentry\captureCheckIn;
+
 class BatchController extends Controller
 {
     use SyncProvider;
 
     public function actionFull()
     {
+        $checkInId = captureCheckIn(
+            slug: Yii::$app->params['sentryMonitorSlug'],
```

Comment:
What is this syntax? I'm not used to that in PHP. It looks like sort of a named-parameter style.

Reply:
I guess I didn't pay enough attention to that code I copied from the Sentry website. I think you're right, though, about it being named parameters. Example: https://stitcher.io/blog/php-8-named-arguments